packages/@sanity/components/src/previews/styles/DefaultPreview.css

```diff
@@ -115,20 +123,25 @@
   @nest & .heading {
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
+    /* justify-content: space-between; */
```

Comment:
can this be removed? if not: make a small comment about why it needs to be there?

Reply:
Noticed it too, removed now